File: 22_max_in_array.c
//C proram to find maximum number amogst user-entered elements in an array

#include <stdio.h>

int main() {
    int num[10];//declared array of numbers
    
    printf("Enter elenemts of array: \n" );
    for(int i=0; i<10; i++){
        printf("Enter element number %d: ", i+1);
        scanf("%d", &num[i]);
    }
    
    //find max
    int max=num[0];
    for(int i=0; i<10; i++){
        if(num[i]>max){
            max=num[i];
        }
    }
    
    printf("Maximum number entered is %d", max);

    return 0;

File: 22_max_in_array.py
#Python program to find maximum number in an array(list)

#number of list elements
n=int(input("Enter number of elements in the array: "))

#declare array and input elements
arr=[]
for i in range(n):
    num=int(input(f"Enter element {i+1}: "))
    arr.append(num)
    
#find and print the maximum number
max_num= max(arr)
print("Maximum of entered numbers is: ", max_num)

        
